search/search_quality/download-maps.sh

```diff
@@ -47,6 +47,8 @@ fi
 
 NAMES=("Australia_Brisbane.mwm"
        "Belarus_Minsk*.mwm"
+       "Canada_Ontario_London.mwm"
```

Comment:
Это нужно для тестов на search quality. Там настоящие mwm и настоящие
запросы. Запросы пойдут отдельным реквестом так как в другой репозиторий.
On Jun 20, 2016 5:17 PM, "Ilya Zverev" <notifications@github.com> wrote:

> In search/search_quality/download-maps.sh
> <https://github.com/mapsme/omim/pull/3609#discussion_r67696879>:
>
> > @@ -47,6 +47,8 @@ fi
> >
> >  NAMES=("Australia_Brisbane.mwm"
> >         "Belarus_Minsk*.mwm"
> > +       "Canada_Ontario_London.mwm"
>
> Это откуда взялось?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mapsme/omim/pull/3609/files/099707924097c8db2d0f68e11b38b226b33024f9#r67696879>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAbvp8BGd4xxcbppIlh1VdnoGhvID2Z1ks5qNqEBgaJpZM4I5sGV>
> .
>